### ISSUE TITLE
Add script alternate in other languages

### ISSUE THREAD

**Issue by kaustubhhiware** [kaustubhhiware/NotiFyre#12]
We can make full use of the inherit advantages of other languages, by adding alternatives in different languages.

Some alternatives off the top could be python, ruby, javascript.

**Comment by kaustubhhiware** [comment]
The python alternate is relatively easy to add.

**Comment by athityakumar** [comment]
I don't understand. Would these alternatives (say, the Python one) actually implement the same things as the bash script on it's own, or just call all the bash script lines with `os`?

**Comment by kaustubhhiware** [comment]
No, there are alternatives including `pynotify` module. It is possible the notification system in some languages does not use the default library (although I'm not sure of it), so let's see.

**Comment by athityakumar** [comment]
Just thought I'd share the resources I gathered for Ruby. 😄 

[This](https://github.com/Codaisseur/terminal-notifier-guard) Ruby gem seems to make it easier for having a Ruby script that takes care of the Mac part (using `terminal-notifier` dependency, I checked - it works). Not sure about [this](https://github.com/cyclotron3k/ruby-notify-send) gem for the `notify-send` part though - it showed me a socket error.
